### ISSUE TITLE
Difference Module  

### ISSUE THREAD

**Issue by bo123bo321** [wgcban/ChangeFormer#113]
Hello, can you introduce the principle of this module

def conv_diff(in_channels, out_channels):
    return nn.Sequential(
        nn.Conv2d(in_channels, out_channels, kernel_size=3, padding=1),
        nn.ReLU(),
        nn.BatchNorm2d(out_channels),
        nn.Conv2d(out_channels, out_channels, kernel_size=3, padding=1),
        nn.ReLU()
    )

**Comment by bo123bo321** [comment]
Why does this module have A similar function as abs (A-B)
